pepper/ext_gadget/pedersen_bridge.cpp

```diff
@@ -56,5 +55,11 @@ protoboard<FieldT> getProtoboard(const char* assignment)
         assert(pb.is_satisfied());
     }
     
-    return pb;
+    return f;
```

Comment:
I am unsure if returning `f` was desired here. Could this be the reason tests aren't passing?

Reply:
The signature changed from returning the protoboard to returning the gadget (since we need to access the gadget's output wires in the unit test). The we also need to change what we are returning.